sorc/build.mpas

```diff
@@ -26,7 +26,7 @@ module load rrfs/${MACHINE}.intel
 module list
 
 cd ${HOMErrfs}/sorc/MPAS-Model
-cp ../_workaround_/MPAS-Model/Registry.xml src/core_atmosphere
+#cp ../_workaround_/MPAS-Model/Registry.xml src/core_atmosphere
```

Comment:
It looks like we no longer need this workaround?
If yes, please remove this line and the Registry.xml file under the `_workaround_` directory. Thanks!